test/checkResult.test.js

```diff
@@ -0,0 +1,144 @@
+// IMPORT MODULES under test here:
```

Comment:
Great job on testing, this is very verbose but honestly you've covered your bases and know your function works the way it's supposed to. Awesome work.